src/main.cpp

```diff
@@ -342,9 +345,6 @@ int main ( int argc, char** argv )
         // HTML status file ----------------------------------------------------
         if ( GetStringArgument ( argc, argv, i, "-m", "--htmlstatus", strArgument ) )
         {
-            qWarning() << qUtf8Printable (
```

Comment:
It seems like some of the latest changes are reverted. The question is how we can have a clean state.